app/pulp/app/models/progress.py

```diff
@@ -18,6 +20,16 @@ class ProgressReport(Model):
 
     All progress reports have a message, state, and are related to a Task.
 
+    Attributes:
+
+        batch_increments (bool): If this Flag is set save checks to make sure it only gets ran once
```

Comment:
I didn't expect any of these would be developer configurable. I think these options will be rarely used if ever. If we keep these as simple as possible then plugin writers will have as little to think about which will make for a better plugin experience. To fix this the feature would be assumed to be always on and `batch_interval` would be hard coded probably as a constant at the top of the module.